src/OrchardCore/OrchardCore/Localization/NullHtmlLocalizerFactory.cs

```diff
@@ -56,7 +56,8 @@ public IEnumerable<LocalizedString> GetAllStrings(bool includeParentCultures)
             public LocalizedString GetString(string name, params object[] arguments) =>
                 NullStringLocalizerFactory.NullLocalizer.Instance.GetString(name, arguments);
 
-            IHtmlLocalizer IHtmlLocalizer.WithCulture(CultureInfo culture) => Instance;
+            [Obsolete("This method will be removed in the upcoming ASP.NET Core major release.")]
```

Comment:
I'm not sure why we need while the Localization APIs already warn about this. Second thing I'm not sure how this fix related to the referred issue, @agriffard is already fixed most if not issues in the PR #6999

Reply:
This pr is not about moving to .Net Core 5 @hishamco - If you read the comments on the pr, it is about enabling usage of OrchardCore nugets in a solution targeting .Net Core 5.

@agriffard pr is about actually moving the target framework to .Net Core 5, which is a different thing.

I put the `Obsolete` in, although not required, so that when we move to .Net Core 5, we will catch it to remove it.

Reply:
> This pr is not about moving to .Net Core 5

Ya, but I'm not sure what's the issue if we didn't add this attribute?

Reply:
It's going to warn those using it that they should stop using this method because it could be eventually be removed.

Reply:
I'm expected ASP.NET Core localization APIs will going to warn insted